src/api/app/components/watchlist_component.rb

```diff
@@ -61,12 +61,15 @@ def toggle_watchable_path
     end
   end
 
+  # Sort projects and packages names in a case-insensitive manner first
+
   def projects
-    @projects ||= Project.joins(:watched_items).where(watched_items: { user: @user }).order(:name)
+    @projects ||= Project.joins(:watched_items).where(watched_items: { user: @user }).order('LOWER(name), name')
   end
 
   def packages
-    @packages ||= Package.includes(:project).joins(:watched_items).where(watched_items: { user: @user }).order('projects.name, packages.name')
+    @packages ||= Package.joins(:project).joins(:watched_items).where(watched_items: { user: @user })
```

Comment:
@saraycp what is the reason to switch from `includes` to `joins`? I think we work with the associated project of the gathered packages here https://github.com/openSUSE/open-build-service/blob/80edd61e9b411c34a052e56702e9e88a18dc9a4f/src/api/app/components/watched_items_list_component.html.haml#L5
So I think using the eager loading of includes is just fine to avoid additional queries

Reply:
@saraycp ahh okay, sorry for the noise, it doesnt work with `includes` in this case...

Reply:
With `includes` the query failed when trying to order by project name. I mean, the problem was on this part `.order('LOWER(projects.name), projects.name')`. I tried both plural and singular...

If you have any better idea, I would appreciate it.